README.md

```diff
@@ -18,14 +18,14 @@ Since Camel-K already does a pretty good job building Camel applications in any
 
 ## How does it work?
 
-Without going into many details, Camel-K has basically two phases of their build, which I call "Project Assemble" and "Image Build". 
+Without going into many details, Camel-K has basically two phases of their build, which I call "Project Assemble" and "Image Build".
 In this first phase, Camel-K reads the Route configuration, assemble the Maven project and run the project build.
 Then it takes the Java application and build into an image.
 
 Camel-K has this concept of "Environment Platform" that based on the type of cluster which is running, can pick the right build feature.
```

Comment:
```suggestion
Camel-K has this concept of "Environment Platform" that is based on the type of cluster in use, so it can pick the right build feature.
```